insights/specs/core3_archive.py

```diff
@@ -14,5 +14,4 @@
 
 class Core3Specs(Specs):
     branch_info = simple_file("/branch_info", kind=RawFileProvider)
-    display_name = simple_file("display_name")
```

Comment:
@SteveHNH leave this here, and add `ansible_host` here as well to support core processing of core-collection archives.